lib/python/pyflyby/_dbg.py

```diff
@@ -292,6 +292,9 @@ def _prompt_continue_waiting_for_debugger():
             count_invalid += 1
         else:
             break
+    print("Exiting after multiple ({}) invalid responses.".format(max_invalid_entries))
+    # Sleep for a fraction of second for the print statements to get printed.
+    time.sleep(0.01)
```

Comment:
I tried 
```
    sys.stderr.flush()
    sys.stdout.flush()
``` 
But still the statements were not getting printed. I believe this is because this is added as a postloop in pdb it is not printing the last messages